code/game/machinery/atmos_exporter.dm

```diff
@@ -93,28 +93,28 @@
 	user.set_machine(src)
 	for(var/obj/machinery/atmos_points_exporter/AE in machines)
 		can_turf = get_turf(AE)
-	var/dat = text("Time until market update: [round((market_cooldown - world.time)/10)] seconds<br>")
+	var/dat = ("Time until market update: [round((market_cooldown - world.time)/10)] seconds<br>")
 	dat += "</div>"
-	dat += text("<A href='?src=\ref[src];action=send'>Export Canister</A> to <A href='?src=\ref[src];action=choose'> [export_name]</A><br>")
-	dat += text("<A href='?src=\ref[src];action=scan'>Scan Canister Contents</A><br><br>")
+	dat += ("<A href='?src=\ref[src];action=send'>Export Canister</A> to <A href='?src=\ref[src];action=choose'> [export_name]</A><br>")

```

Comment:
Do you have an example of making a ton of text and HTML into a list, I WILL screw it up the first 12 times otherwise.